src/exchanges/ccxt/index.ts

```diff
@@ -391,7 +391,7 @@ export default class CCXTExchangeWrapper implements PublicExchangeAPI, Authentic
         const sourceSymbol = await this.getSourceSymbol(symbol);
         try {
             const rawTrades = await this.instance.fetchTrades(sourceSymbol, since, limit, params);
-            return rawTrades.map(({info, id, timestamp, datetime, symbol: _symbol, order, type, side, price, amount}) => ({
+            return rawTrades.map(({id, timestamp, symbol: _symbol, side, price, amount}) => ({
```

Comment:
`_symbol` is a bit misleading here, even though it was already like this.

Reply:
@fb55 Thanks, good point. Renamed _symbol to productId to match where it is used below.